docs/src/components/frame/frame.tsx

```diff
@@ -53,6 +53,9 @@ export const Frame: React.FC<Props> = props => {
     [styles.light]: themeName === Theme.names.light
   })
 
+  const isExamplePage = window.location.pathname.includes('examples')
+  if (isExamplePage) return <ExamplePage>{children}</ExamplePage>
+
```

Comment:
this feels so hacky...but i couldn't get other approaches to work

Reply:
Needs a canUseDOM() protection for SSR.